[ADD] Attendances: moving check in section to new doc (#9434)
odoo/documentation: content/applications/hr/attendances.rst

Access rights
=============

> It is important to understand how the different access rights affect what users can access the
> *Attendances* application and its features.

Reviewer: ```suggestion
It is important to understand how the different access rights affect which features users can access in the
*Attendances* application.
```
